website/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["next/babel"], "@emotion/babel-preset-css-prop"],
+  "presets": [["next/babel"], ["@babel/preset-react", { "runtime": "automatic" }]],
```

Comment:
Not sure where to add this comment, but it looks like the prop tables disappeared
![Screen Shot 2022-04-11 at 2 54 28 PM](https://user-images.githubusercontent.com/2414030/162809958-f20e4e97-dadc-445b-95f1-2b3af6b016e9.png)

Reply:
I never noticed that `remark` might have [stopped](https://github.com/remarkjs/remark/discussions/635#discussioncomment-390722) supporting `table` markdown types from <=12 onwards. I've added `remark-gfm` as a plugin to bring the functionality back here.